src/pages/dashboard/components/ContentOverviewProject.tsx

```diff
@@ -145,7 +145,7 @@ const ContentOverviewProject = () => {
               column.header === "Hectares" ? (column.header = "Restoration Hectares") : column.header;
               return {
                 ...column,
-                enableSorting: true
+                enableSorting: column.header?.length === 0 ? false : true
```

Comment:
Instead of checking if `column.header?.length === 0` and returning `false`, you can directly use the logical negation. something like this:
 `enableSorting: Boolean(column.header?.length)`